ReadMe.md

```diff
@@ -166,6 +166,40 @@ We are primarily considering JavaScript (sorry, not TypeScript) as the base lang
 
 ## FAQs
 
+#### About OpenSource Projects
```

Comment:
Make it in a question way, and also, put this as a Level 3 heading. If not, add another Level 3 heading above it, saying General FAQs.

Reply:
Okay sir